java/arcs/core/storage/BUILD

```diff
@@ -74,3 +80,13 @@ arcs_kt_library(
         "//java/arcs/core/crdt",
     ],
 )
+
+arcs_kt_library(
+    name = "writeback",
+    srcs = WRITEBACK_SRCS,
+    deps = [
+        "//java/arcs/core/storage/keys:protocols",
+        "//third_party/java/androidx/annotation",
```

Comment:
This AndroidX library probably won't work here, `arcs.core` is intended for cross-platform Kotlin only, and I don't think the androidx annotations are safe for compilation to JS (for example).

Reply:
ah, I didn't consider "cross-platform" :(
Will figure out a way of moving out of arcs.core